docs/data/base/components/switch/switch.md

```diff
@@ -22,23 +21,45 @@ To make the switch accessible, you should ensure the corresponding label reflect
 import SwitchUnstyled from '@mui/base/SwitchUnstyled';
 ```
 
-The `SwitchUnstyled` component provides default components and assigns CSS classes you can style entirely on your own.
-You are free to choose any styling solution - plain CSS classes, a CSS framework, Emotion, etc.
-It is also possible to replace these default components by other HTML elements or custom components.
+## Basic switch
+
+The `SwitchUnstyled` component is composed of a root `<span>` that houses three interior elements—a track, a thumb, and an input—and it assigns CSS classes for styling each piece:
+
+```html
+<span class="MuiSwitch-root Mui-checked css-i6datd">
```

Comment:
There's no need to include class names generated by emotion to illustrate the structure of the component.

```suggestion
<span class="MuiSwitch-root Mui-checked">
```